src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/ProjectsAjaxController.java

```diff
@@ -21,25 +23,42 @@
 @RestController
 @RequestMapping("/ajax/projects")
 public class ProjectsAjaxController {
-	private final UIProjectsService UIProjectsService;
-
+	private final UIProjectsService projectsService;
+	private final MessageSource messageSource;
 
 	@Autowired
-	public ProjectsAjaxController(UIProjectsService UIProjectsService) {
-		this.UIProjectsService = UIProjectsService;
+	public ProjectsAjaxController(UIProjectsService UIProjectsService, MessageSource messageSource) {
+		this.projectsService = UIProjectsService;
+		this.messageSource = messageSource;
+	}
+
+	/**
+	 * Create a new project.
+	 *
+	 * @param request {@link CreateProjectRequest} containing the name for the project and any other details
+	 * @param locale {@link Locale} for the currently logged-in user
+	 * @return Response will contain the created project identifier, or an error message
+	 */
+	@PostMapping("/new")
+	public AjaxResponse createNewProject(@RequestBody CreateProjectRequest request, Locale locale) {
+		try {
+			return new AjaxCreateItemSuccessResponse(projectsService.createProject(request));
+		} catch (Exception e) {
+			return new AjaxErrorResponse(messageSource.getMessage("server.CreateProject.error", new Object[]{}, locale));
+		}
 	}
 
 	/**
 	 * Handle request for get a filtered and sorted list of projects for a user or administrator
```

Comment:
Change `get` to `getting`

Reply:
Updated in 22eea37ca51bc35fa2f8f199f79cb024e0099235